ddtrace/contrib/algoliasearch/patch.py

```diff
@@ -12,29 +10,59 @@
 APP_NAME = 'algoliasearch'
 SEARCH_SPAN_TYPE = 'algoliasearch.search'
 
-# Default configuration
-config._add('algoliasearch', dict(
-    service_name=SERVICE_NAME,
-    collect_query_text=False
-))
+try:
+    import algoliasearch
+    from algoliasearch.version import VERSION
+    algoliasearch_version = tuple([int(i) for i in VERSION.split('.')])
+
+    # Default configuration
+    config._add('algoliasearch', dict(
+        service_name=SERVICE_NAME,
+        collect_query_text=False
+    ))
+except ImportError:
+    algoliasearch_version = (0, 0)
 
 
 def patch():
+    if algoliasearch_version == (0, 0):
```

Comment:
This shouldn't be needed since we do version checks below.

But also doesn't hurt anything to keep in here.